modules/oe_webtools_page_feedback/src/Form/WebtoolsPageFeedbackSettingsForm.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_page_feedback\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides configuration form for the Page Feedback Form webtools widget.
+ */
+class WebtoolsPageFeedbackSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId(): string {
+    return 'oe_webtools_page_feedback_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state): array {
+    $config = $this->config('oe_webtools_page_feedback.settings');
+
+    $form['enabled'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enabled'),
+      '#description' => $this->t('Check this box if you would like to enable the Page feedback form on all the content pages.'),
+      '#default_value' => $config->get('enabled'),
+    ];
+    $form['formID'] = [
+      '#type' => 'number',
+      '#title' => $this->t('Form ID'),
+      '#description' => $this->t('Provide your webtools form ID.'),
```

Comment:
I'm wondering if we shouldn't provide more info here on how to request the ID... Let's leave it like this and we can fix it if we need later in the epic.

Reply:
This is how you request a service https://webgate.ec.europa.eu/fpfis/wikis/display/webtools/Default+Feedback+Form#DefaultFeedbackForm-Howtorequestthisservice and there is a link in the [README](https://github.com/openeuropa/oe_webtools/pull/168/files#diff-cfffa78681c6bd04aee6f8351dd5c3852589cdfb8ab5b1c791768fe0524ce886R9) pointing to the webtools widget page so developers can find the information. For ewcms only CEM should have access (they would know already the form id).

Do you think it's still necessary?

Reply:
Good point, CEM should already be aware of this so lets leave it like this.

Reply:
We are in a component here, so CEM is besides the point. But yea, we link in the readme to webtools so that should be fine.